communication/slack-config/sig-cloud-provider/config.yaml

```diff
@@ -1,5 +1,19 @@
 channels:
   - name: apiserver-network-proxy
   - name: cloud-provider-extraction
+  - name: provider-aws
+    id: C0LRMHZ1T
+  - name: provider-azure
+    id: C5HJXTT9Q
+  - name: provider-gcp
+    id: C6WB33KNJ
+  - name: provider-ibmcloud
+    id: C90BWDWCE
+  - name: provider-openstack
+    id: C0LSA3T7C
+  - name: provider-vsphere
+    id: C9PGCDKV5
+  - name: provider-centurylink
+    id: C0P8MQUM9
```

Comment:
More so an FYI - You only need to provide the channel `id` if it is being renamed 👍 
...I should have mentioned that in the original PR when moving `sig-cloud-provider` over. 🙈 

 A small nit and I wouldn't consider it a blocker, but figure while moving them all over, might as well alpha sort?

Reply:
So the ids (other than centrurylinks) were transferred from the source. 
It wasn't clear to me if they had been renamed, so I thought it was safer to just leave them.

I went ahead and alpha sorted them